src/screens/SettingsScreen.js

```diff
@@ -2,6 +2,8 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { withHandlers } from 'recompact';
 import styled from 'styled-components/primitives';
+
+import SendFeedback from "../components/SendFeedback";
```

Comment:
move to line 10 to be `abc` alphabetical

Reply:
👍